src/styles/blocks/calendar.scss

```
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+  max-width: calc(#{$width + 7} * 7);
+  gap: 1px;
+
+  @for $index from 28 through 31 {
+    &--month-length-#{$index} {
+      & :nth-child(n + #{$index + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    padding: 10px;
+    background-color: #eee;
+    border: $border-width solid black;
+    height: 100px;
+    width: $width;
+    text-align: center;
+    cursor: pointer;
+    transition-duration: 0.5s;
+
+    &:hover {
+      background-color: pink;
+      transform: translate(0, -20px);
+    }
+
+    @for $count from 1 through 31 {
+      &:nth-child(#{$count})::before {
+        content: "#{$count}";
+      }
+    }
+  }
+}
+
+.page__calendar {
+  margin: 10% auto;
+}
```

Move this code to separate file. Use nesting.
```suggestion
.page {
  &__calendar {
    margin: 10% auto;
}
```